src/modules/guide-steps.js

```diff
@@ -6,13 +6,15 @@ const smallScreenWidth = 640;
 
 export function nextStep() {
 	currentStep++;
+	getJQ( '#dmguide .dmguide-step' ).html( getStepHtml() );
```

Comment:
Sorry for another request. The `getJQ` simply returns `jQuery` function and accepts no arguments. So this line should be `getJQ()( ''#dmguide .dmguide-step'' )`.

Reply:
Where does `getJQ()` come from?